tmp/build/manifest-templates/clusterserviceversion.yaml

```diff
@@ -69,15 +69,15 @@ metadata:
                 "traceSampling": 100.0
               },
               "kiali": {
-                "enabled": false
+                "enabled": true
               },
               "grafana": {
-                "enabled": false
+                "enabled": true
               },
               "tracing": {
                 "enabled": true,
                 "jaeger": {
-                  "template": "all-in-one"
+                  "template": "production-elasticsearch"
                 }
```

Comment:
This seems to be a build artifact and should not be checked into git, can we also clean this up and set up .gitignore?

Reply:
Never mind, I had assumed this was being created from the ones above but it looks as if this is a template for build and the other ones are ignored